playbook-website/app/javascript/components/Icons/index.tsx

```diff
@@ -1,7 +1,7 @@
 import React from "react"
 
 import { Icon, Title, Flex, FlexItem, Card } from "playbook-ui"
-import Roofing from "!babel-loader!react-svg-loader!@powerhome/playbook-icons/icons/roofing.svg"
+import Roofing from "@powerhome/playbook-icons/icons/roofing.svg"
```

Comment:
Here, we are testing the import of an SVG React element in the view. It is working well.